core/templates/components/forms/custom-forms-directives/html-select.component.html

```diff
@@ -1,15 +1,15 @@
 <div uib-dropdown class="oppia-html-select">
```

Comment:
Can you try and use ngb-dropdown here instead of uib-dropdown. uib is only angularjs. https://ng-bootstrap.github.io/#/components/dropdown/examples

Reply:
@srijanreddy98, could you please tell me on which page this `html-select` dropdown is located?